controllers/FileUploadController.js

```diff
@@ -0,0 +1,83 @@
+const fileUploadModel = require("../models/FileUploadModel");
+const apiResponse = require("../helpers/apiResponse");
+const { GridFsStorage } = require("multer-gridfs-storage");
+const mongoose = require('mongoose');
+const multer = require("multer");
+const { CONFIG } = require("../helpers/config");
+const auth = require("../middlewares/jwt");
+const { body, cookie } = require("express-validator");
+const Grid = require('gridfs-stream');
+Grid.mongo = mongoose.mongo;
+const gfs = Grid(mongoose.connection);
+gfs.collection('files');
+
+
+// DB connection
+const MONGODB_URL = CONFIG.MONGODB_URL;
+
+const storage = new GridFsStorage({
+  url: MONGODB_URL,
+  file: (req, file) => {
+    return {
+      bucketName: "files",
+      //Setting collection name, default name is fs
+      filename: file.originalname,
+      //Setting file name to original name of file
+    };
+  },
+});
+
+storage.on("connection", (db) => {
+  //Setting up upload for a single file
+  upload = multer({
+    storage: storage,
+  }).array("files", 5);
+});
+
+exports.uploadFiles = [
+  auth,
+  (req, res) => {
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 3 tabs but found 6 spaces.](https://app.codacy.com/gh/n4alphapesto/hirebook-backend/pullRequest?prid=7771833)
```suggestion
			fileIds
```